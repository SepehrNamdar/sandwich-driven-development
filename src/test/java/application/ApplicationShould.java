package application;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class ApplicationShould {

    @Test
    void works() {
        Application application = new Application();
        assertThat(application.isWorking()).isTrue();
    }
}
