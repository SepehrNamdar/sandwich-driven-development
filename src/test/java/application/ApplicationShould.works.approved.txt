[123, 2021-02-20] => Interview{candidate=Candidate{skills=[.Net, Java, PHP, JS]}, recruiter=Recruiter{skills=[Java, .Net, PHP, JS]}, interviewDate=2021-02-20} 
[null, 2021-02-20] => Interview{candidate=Candidate{skills=[.Net, Java, PHP, JS]}, recruiter=Recruiter{skills=[Java, .Net, PHP, JS]}, interviewDate=2021-02-20} 
